app/views/volunteers/show.html.slim

```diff
@@ -124,6 +124,12 @@ h3 Persönlicher Hintergrund
       tr
         td #{t_attr(:external)} / #{t_attr(:internal)}
         td= @volunteer.external? ? t_attr(:external) : t_attr(:internal)
+        tr
```

Comment:
you have a wrong indent here. It needs to be one level less deep, because this way the row is a child of the tr at line 124